build/Build.proj

```diff
@@ -0,0 +1,44 @@
+<Project DefaultTargets="Build;Test;Pack">
```

Comment:
I would not move this to a less discoverable location, for the reasons metioned in the previous comment. I would also make this a corebuild.io standard project and get rid of the custom restore.

Reply:
Right, agreed. If `build.cmd` indeed is allowed to go away, there should be a `build.proj` in the root.